apps/site/components/EOL/EOLReleaseTable/index.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
```

Comment:
This shouldn't be a client component, does it really need to be? It forces the provdeVulnerabilities and provideReleaseData to run on client-side too.

+ Even if EOLModal is a client component, that's completely fine.

Reply:
I see, that's due to useState.... 🤔 

In this case, I wonder how this will behave. It will probably run these fetch() requests on the client-side, which we want to avoid.

Reply:
<img width="207" height="159" alt="image" src="https://github.com/user-attachments/assets/44fcfef6-e5ae-4213-a24e-9ad30101361d" />

Yeah it is requesting on every load, although it does then cache and load from disk cache. My wonder is:

- Knowing all is behind CDN, it shouldn't be an issue, and it fetches just in case the state is different, which would trigger a re-render
  - This could though create a situation of React complaining server content !== client content
  - But this should be fine due to skew protection and worst case, just re-renders.

Reply:
The only thought here is, do we have other pages that are also running these fetch requests on client-side? During the development of this website, we made a conscious effort for all these requests to only be server-side, should we be fine with them being client-side? Instead of having mixed JSONs within the bundle?